docker/docker-compose.yml

```
@@ -82,6 +88,12 @@ services:
       timeout: 10s
       retries: 3
       start_period: 20s
+    logging:
+       driver: "awslogs"
+       options:
+          awslogs-region: "us-east-1"
```

```suggestion
          awslogs-region: "us-east-1"
```

---

Sorry, this was meant to be:

```suggestion
          awslogs-region: "${AWS_DEFAULT_REGION}"
```